console-subscriber/src/builder.rs

```diff
@@ -101,4 +104,41 @@ impl Builder {
     pub fn build(self) -> (TasksLayer, Server) {
         TasksLayer::build(self)
     }
+
+    /// Configures this builder from a standard set of environment variables:
+    ///
+    /// * `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+    ///   completed tracing data. Default: 60s.
+    /// * `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+    ///   localhost:1234 or similar. Default: 127.0.0.1:6669
+    /// * `TOKIO_CONSOLE_PUBLISH_INTERVAL`: The number of milliseconds to wait
+    ///   between sending updates to the console. Default: 1000ms (1s)
+
+    pub fn from_default_env(mut self) -> Self {
+        if let Ok(retention) = std::env::var("TOKIO_CONSOLE_RETENTION") {
+            self.retention = Duration::from_secs(
+                retention
+                    .parse()
+                    .expect("TOKIO_CONSOLE_RETENTION must be an integer"),
+            );
+        }
```

Comment:
since we have two different duration values that are configured from an env var, and they're in different units, i think we ought to make the env var names include the unit. it's good that we list the units in the docs, but i think it's probably good to also ensure that it's clear when you see the variables _used_ --- you might be, for example,  reading a shell script that says something like

```bash
TOKIO_CONSOLE_RETENTION=60 TOKIO_CONSOLE_PUBLISH_INTERVAL=1000 ./myapp
```
and get the wrong impression that the retention period is significantly shorter than the publish interval.

so, i'd suggest:
```suggestion
        if let Ok(retention) = std::env::var("TOKIO_CONSOLE_RETENTION_SECS") {
            self.retention = Duration::from_secs(
                retention
                    .parse()
                    .expect("TOKIO_CONSOLE_RETENTION_SECS must be an integer"),
            );
        }
```

Reply:
Do you think it would make sense to use minutes for the retention period? Is it likely people will need <1min retention?